src/back/interfaces/handlers.py

```diff
@@ -0,0 +1,14 @@
+"""Module containing handlers' interfaces."""
+from typing import Generic, Self, TypeVar
+
+from src.back.interfaces.io import WriteStreamInterface
+
+T = TypeVar("T")
+
+
+class MessageHandlerInterface(Generic[T]):
+    """Message handler interface."""
+
+    def handle(self: Self, message: T, stream: WriteStreamInterface) -> None:
```

Comment:
3) Here I am saying that the type I pass to generic interface, will be used for message argument

Reply:
Yes